### PR TITLE
Test with webhippie/elasticsearch:6.4

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
acceptance tests are failing because files cannot be found (searching for file content)

local tests of these scenarios show that files are found as soon as cron was executed successfully.

no idea how this could make a difference ....

File: .drone.star [hunk 1]
```diff
@@ -75,7 +75,7 @@ config = {
 			'extraServices': [
 				{
 					'name': 'elasticsearch',
-					'image': 'webhippie/elasticsearch:5.6',
```
Comment (DeepDiver1975) [inline .drone.star:78]: @phil-davis I'd like to run all tests with both version (later even more) - can I ask you to apply your drone-fu here? THX a lot

**Comment by DeepDiver1975** [comment]
> local tests of these scenarios show that files are found as soon as cron was executed successfully.

I wonder if the scenarios are green if we add a cron call in the before step of WebUIElasticSearchContext

**Comment by phil-davis** [comment]
Assigned myself so that I see this later today...

**Comment by DeepDiver1975** [comment]
> Assigned myself so that I see this later today...

:wave: - just to make sure this is not forgotten ;-)

**Comment by micbar** [comment]
closing. We use `owncloudci/elasticsearch:latest`

**Pull request closed without merging**